joystick_control_node:
  ros__parameters:
    twist_topic: "/diff_drive_controller/cmd_vel_unstamped"
    max_linear_speed: 1.5
    min_linear_speed: 0.0
    max_angular_speed: 0.7
    min_angular_speed: 0.0
    timer_frequency: 0.1
    diag_timer_frequency: 1.0
    min_joy: -1.0
    max_joy: 1.0
    deadzone_threshold: 0.1
    axis_angular: 3
    axis_linear: 4
    button_stop: 0
    button_start: 2
